src/entropymode.rs

```diff
@@ -0,0 +1,704 @@
+#![allow(non_upper_case_globals)]
```

Comment:
Can you please add a header to match (most of) the other source files?

Reply:
Yep, good catch. Done.

Reply:
Just nitpicking, but there's an extra newline at the start of this file.

Reply:
ah so there is!

Reply:
Thanks. :+1: